setup.cfg

```diff
@@ -1,5 +1,6 @@
 [metadata]
 name = gnocchi
+version = 4.5
```

Comment:
you might want to use `setuptools_scm`

Reply:
Thanks! I should've added that when porting it, solved now.